matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/query/LiveLocationShareAggregatedSummaryEntityQuery.kt

```diff
@@ -55,3 +55,11 @@ internal fun LiveLocationShareAggregatedSummaryEntity.Companion.getOrCreate(
     return LiveLocationShareAggregatedSummaryEntity.where(realm, roomId, eventId).findFirst()
             ?: LiveLocationShareAggregatedSummaryEntity.create(realm, roomId, eventId)
 }
+
+internal fun LiveLocationShareAggregatedSummaryEntity.Companion.get(
+        realm: Realm,
+        roomId: String,
+        eventId: String,
+): LiveLocationShareAggregatedSummaryEntity? {
+    return LiveLocationShareAggregatedSummaryEntity.where(realm, roomId, eventId).findFirst()
+}
```

Comment:
Not for this PR, but now the line 55 could be replaced by a call to this new method.